src/Map/Map.js

```diff
@@ -60,6 +57,16 @@ class Map extends React.Component {
     // setup select interactions for the map
     this.initializeSelect(this.map)
 
+    // callback for <Provider> if it is mounted as hoc
+    const mapContext = {
+      map: this.map,
+      mapId: this.target,
+      selectInteraction: this.selectInteraction,
+      translations // this can be hoisted to <Provider> only in the future
```

Comment:
add `index: this.props.index`

Reply:
save this for future splitscreen PR

Reply:
actually- index should probably just be assigned and managed by `<Provider>` & `<Map>` doesn't need to know about it